battleship.py

```diff
@@ -8,7 +8,17 @@ def place_ships():
 
 
 def get_target():
-    pass
+    choice = input("Where would you like to aim? (example: '2,3')")
+    if len(choice) > 3:
```

Comment:
The problem with checking if it's length is greater than 3 is if we have a 16x16 grid and they enter something like [12,13] which is a valid coordinate it will not work. 

This is fine for now we will fix it later. No further work needed.